spec/jobs/request_job_spec.rb

```diff
@@ -18,7 +18,7 @@
       ActiveJob::Base.queue_adapter = :test
```

Comment:
Useless test because it will never break. Accept it as a rule, there is no need to test rails functionality in your application tests.